docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  btfs-build-binaries:
+    build:
+      context: .
+      dockerfile: Dockerfile.unit_testing
+    volumes:
+      - ../btfs-binary-releases:/btfs-binary-releases
```

Comment:
This looks like a path on jenkins, have you tested docker-compose manually on there?

Reply:
i tested it by running the jenkins job on my dev branch and it looked fine:
https://builds.bittorrent.com/job/BTFS_Build_Binary/106/